src/views/validated-designs/types.ts

```diff
@@ -4,7 +4,7 @@ export interface HvdCategoryGroup {
 	slug: string // start of slug e.g. terraform-operation-guides-adoption, used to add to it's guides after the category group is created
 	title: string // from YAML
 	description: string // from YAML
-	product: ProductSlug
+	product: Exclude<ProductSlug, 'sentinel'>
```

Comment:
had to update this because `<IconTileLogo size="large" productSlug={category.product} />` was complaining about the type for `productSlug`